src/services/lnd/utils-bos.ts

```diff
@@ -0,0 +1,91 @@
+import { readFile } from "fs"
+
+import { baseLogger } from "@services/logger"
+
+import { getChannels, GetChannelsResult } from "lightning"
+
+import { pushPayment, reconnect } from "balanceofsatoshis/network"
+
+import { simpleRequest } from "balanceofsatoshis/commands"
+
+import { getLnds } from "./utils"
+
+import { LndService } from "."
+
+export const reconnectNodes = async () => {
+  const lndService = LndService()
+  if (lndService instanceof Error) throw lndService
+
+  const lnds = getLnds({ type: "offchain", active: true })
+
+  for (const lnd of lnds) {
```

Comment:
Why did we change these lines away from:
```ts
const lndsParamsAuth = getLnds({ type: "offchain" })
for (const { lnd } of lndsParamsAuth) {
  await reconnect({ lnd })
}
```

It looks like `reconnect` takes a `{ lnd: <Authenticated LND gRPC API Object> }` argument.

Reply:
yup. you've catched a bug! maybe that's why reconnect was flacky 😒